sdk/ml/azure-ai-ml/tests/recordings/dsl/e2etests/test_dsl_pipeline_on_registry.pyTestDSLPipelineOnRegistrytest_pipeline_with_local_component_and_registry_model_as_input.json

```diff
@@ -0,0 +1,818 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://eastus.api.azureml.ms/registrymanagement/v1.0/registries/testFeed/discovery",
```

Comment:
If testFeed is having issue, let's use sdk-canary

Reply:
I think it's not problem of testFeed. It's a record problem of registry. It seems all registry relevant tests have been skip if in playbackmode.
![image](https://user-images.githubusercontent.com/26239730/197149906-975290c6-5ee8-40c2-a125-c2021521893d.png)

Reply:
I think @elliotzh has mentioned this issue. but this workaround doesn't work for registry
![image](https://user-images.githubusercontent.com/26239730/197150133-bf29f6a3-a229-412e-87e3-306feaa8aac1.png)